website/docs-scraper/index-website-content.mjs

```diff
@@ -186,11 +202,11 @@ async function indexWebsiteContent() {
     const matches = [...matchAllResults];
 
     // console.log('========================================');
-    console.log(chalk.white(`Processing file: ${fileRelativePath}`));
+    console.log(`Processing file: ${fileRelativePath}`);
 
     if (matches.length) {
       // extract from each "section" the tab name and the actual content
-      matches.forEach(async (match) => {
+      for (const match of matches) {
```

Comment:
This was a silly error by me: essentially I was calling an async function within a `forEach` loop (🤦‍♂️) and this was causing the `markdownRecords` to be populated with different content even after the `forLoop` was executed and the effect was that the `markdownRecords.length` changed at different times in the execution of the code **_after_** the markdown was already processed.